samples/kubeflow-tf/kubeflow-training-classification.py

```diff
@@ -102,7 +102,7 @@ def kubeflow_training(output, project,
   # TODO: use the argo job name as the workflow
   workflow = '{{workflow.name}}'
   # set the flag to use GPU trainer
-  use_gpu = True
+  use_gpu = False
```

Comment:
Why change the use_gpu to false?

Reply:
By default, the gpu is not enabled in kubeflow's deployment. I am going to add a doc about how to enable gpu in new/existing cluster and will put the doc link in comment here.

Reply:
SG